hw2/04-background.js

```diff
@@ -1 +1,64 @@
-// Add your code here
+// Stuart Rimel
+const bodyElement = document.getElementsByTagName('body');
+const intervalInputElement = document.getElementById('intervalInput');
+const buttonElement = document.getElementById('input-btn');
+const ALPHA = 0.7;
+let interval = '';
+let intervalID = null;
+
+const getRandomColor = function getRandomColor() {
+  const r = Math.floor(Math.random() * 255);
+  const g = Math.floor(Math.random() * 255);
+  const b = Math.floor(Math.random() * 255);
+  return `rgba(${r}, ${g}, ${b}, ${ALPHA})`;
+};
+
+const setRandomBackgroundColor = function setRandomBackgroundColor() {
+  bodyElement[0].style.backgroundColor = getRandomColor();
+};
+
+const handlePageLoad = function handlePageLoad(event) {
+  setRandomBackgroundColor();
+  buttonElement.disabled = true;
+};
+
+const isValidInput = function isValidInput(input) {
+  if (input.length > 0) {
+    if (!input.includes('-') && !input.includes('.')) {
+      return true;
+    }
+  }
+  return false;
+};
```

Comment:
_Logic_ - This is great, and I love that you disable the button for invalid input! One thing I noted is that you can still set the value to 0 and have a strobe effect, which I now know is something that I want to check for in my code too.

Reply:
I kind of liked the strobe effect and thought of it as a feature lol. But I guess that it's incorrect? To be safe I went ahead and re-worked it in updated PR

Reply:
I think someone said in lecture it could trigger an epileptic seizure if it's too fast, that's the only reason I mentioned it.